cognee/tests/integration/test_relational_db_migration.py

```
@@ -0,0 +1,137 @@
+                target_props = json.loads(m_data["properties"])
+
+            source_name = normalize_node_name(source_props.get("name", f"id:{n_data['id']}"))
+            target_name = normalize_node_name(target_props.get("name", f"id:{m_data['id']}"))
+
+            found_edges.add((source_name, target_name))
+            distinct_node_names.update([source_name, target_name])
+
+    elif graph_db_provider == "networkx":
+        nodes, edges = await graph_engine.get_graph_data()
+        node_map = nodes_dict(nodes)
+        for (src, tgt, key, edge_data) in edges:
```

Ruff (0.8.2)

Loop control variable `edge_data` not used within loop body

Rename unused `edge_data` to `_edge_data`

(B007)